cdap-examples/WordCount/src/main/java/co/cask/cdap/examples/wordcount/WordCount.java

```diff
@@ -16,34 +16,86 @@
 
 package co.cask.cdap.examples.wordcount;
 
+import co.cask.cdap.api.Config;
 import co.cask.cdap.api.app.AbstractApplication;
 import co.cask.cdap.api.data.stream.Stream;
 import co.cask.cdap.api.dataset.lib.KeyValueTable;
 import co.cask.cdap.api.dataset.table.Table;
 
 /**
- * Word count sample Application.
+ * Word count sample Application. Uses a configuration class which can be used to pass in a configuration during
```

Comment:
Word count sample Application. Includes a configuration class which can be used to pass in a configuration during
application deployment time. [consistent with other usages]